### PR TITLE
feat: home carousel

### PR DESCRIPTION
**Pull request by simboonlong**
## Summary

- add carousel to home page landing section

## Changes

- added swiper js
- setup admin config.yaml for carousel
- refactored welcome section

## Issues

- https://www.pivotaltracker.com/story/show/183176763

### CODE REVIEW

File: cms/carousel/home-carousel.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      label: Verify Documents
+      route: /verify
+    backgroundImage: /static/uploads/home-map.png
+  - title: White Paper
+    subheader: UN/CEFACT White Paper eDATA Verifiable Credentials for Cross Border Trade
+    description: Learn more about this technology method that underpins how
+      TradeTrust deals with verifiable documents!
+    buttonDownload:
+      label: Download Now
+      file: /static/uploads/imdatradetrust_newsletter_issue-06.pdf
```
Comment (simboonlong) [inline cms/carousel/home-carousel.md]: need to get the whitepaper pdf later

File: public/admin/config.yml [hunk 2]
```diff
@@ -153,3 +153,72 @@ collections:
...
+                name: "subheader"
+                widget: "string"
+                required: false
+              - label: "Description"
+                name: "description"
+                widget: "text"
+              - label: "Background image"
+                name: "backgroundImage"
+                widget: "file"
+                required: false
```
Comment (cavacado) [inline public/admin/config.yml:182]: make required instead? then no need to do the check in the component

**Review by cavacado — COMMENTED** [review]
Looks good, but question, is the carousel supposed to transition after x number of secs automatically or no?

File: public/admin/config.yml (hunk 2 above)
Comment (simboonlong) [reply public/admin/config.yml:182]: just background image only? i was thinking of keeping optional so can still publish without it

Comment (cavacado) [reply public/admin/config.yml:182]: ah okok

**Review by cavacado — APPROVED** [review]

**Pull request merged**